source/edge-server/manage-edge-servers.txt

```diff
@@ -0,0 +1,485 @@
+
+There are two types of services that enable you to use Edge Server:
+
+- Edge Server Service: use this option if you want clients to connect to the 
+  Edge Server with MongoDB Drivers and tools. It infers and updates the schema 
+  from the documents you insert, and provides a developer experience familiar 
+  to MongoDB developers. The Edge Server Service rejects incoming Atlas Device
+  SDK connections.
+- Device Sync Service, adding Edge Servers: use this option if you want to 
+  develop client apps using Atlas Device SDK. This option involves more strict 
+  schema enforcement, which the SDKs use to automatically sync objects 
+  across devices and handle conflict resolution.
+
+Regardless of which type of service you choose, Edge Servers use Device Sync 
+to sync data between the Edge Server instance and Atlas. Even when you choose 
+an Edge Server Service, you must first enable Device Sync before you can create 
+Edge Server instances.
+
+.. important:: You can't add Atlas Device SDK Apps later to an Edge Server Service
+
+   You can always add Edge Servers to a Device Sync Service. But you can't use
+   Atlas Device SDK connections with an Edge Server Service. 
+   
+   Users are not portable between services. If you start with an Edge Server 
+   Service, but later want to add an Atlas Device SDK application, there is no 
+   way to transfer any users in your Edge Service to a new Device Sync Service.
+
+You can create an Edge Server-compatible service in Data Services or App 
+Services. 
+
+- You might select Data Services when you just want to get started building
+  with Edge Server or Device Sync and Edge Server, and don't need other 
+  App Services.
+- You might select App Services when you want to bundle your service with 
+  authentication, Functions, Triggers, HTTPS Endpoints, or Values and Secrets.
+
+.. tabs::
+
+   .. tab:: Create a Data Service
+      :tabid: atlas-service
+
+      .. include:: /includes/create-service-data-services.rst
+
+      .. include:: /includes/edge-server-enable-device-sync.rst
+
+   .. tab:: Create an App Service
+      :tabid: app-service
+
+      Refer to the documentation to :ref:`create-app`.
+
+      When you choose your App type, select ``Edge Server`` or ``Device Sync``. 
+
+      .. include:: /includes/edge-server-enable-device-sync.rst
+
+Supported Authentication Providers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Edge Server supports a subset of authentication providers:
+
+- :ref:`anonymous-authentication`
+- :ref:`email-password-authentication`
+- :ref:`custom-jwt-authentication`
+- :ref:`api-key-authentication`
+
+You must enable at least one of these providers to enable clients to connect
+to the Edge Server.
+
+.. _create-edge-server-instance:
+
+Create an Edge Server Instance
+------------------------------
+
+After creating an Edge Server Service, you can add Edge Servers to it. The
+service can handle multiple Edge Servers, each of which has its own 
+registration key.
```

Comment:
Do we want to say its own API key here? That is what the Edge Server actually uses to authenticate with the Cloud, the registration key is only used to retrieve the API key. But also, the customer will only see the registration key and not the API key so curious what you think would be better?

Reply:
Yeah, since registration key is the language used in the UI, that's the language I've chosen here for the docs. On the "Configure" page, I cover that the registration key is exchanged for an API Key.

I don't know how much into the weeds we want to get around these details, so I'll defer to you and Sudarshan about that. If it seems important to use "API Key" for security purposes or otherwise, I'm happy to change it.

Reply:
Yeah I think this is good as is then, thanks!